### PR TITLE
WIP: Add fosuserbundle recipe.

### PR DESCRIPTION
**Pull request by kporras07**
Q | A
-- | --
License | MIT

Hi there,

I'd like to help to create the fosuserbundle recipe.

Right now, I have 2 blockers:
- I don't know how to test the recipe locally, so, I just tried to add the things that I think are right, but I can't test it.
- I don't know how to configure security (https://symfony.com/doc/master/bundles/FOSUserBundle/index.html#step-4-configure-your-application-s-security-yml) I mean, it's not the bundle config file, it's a file from another bundle.

I'd appreciate any help and guidance you can provide me.

Btw, thanks for creating such great tools!

### CODE REVIEW

**Comment by kporras07** [comment]
I was making this for a hobby and learning how recipes work. During DrupalCon I talked to @weaverryan and @nicolas-grekas and this one is most likely to not pass; so, feel free to close it if it's convenient :)

File: friendsofsymfony/user-bundle/2.1/src/Repository/UserRepository.php [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+class UserRepository extends ServiceEntityRepository
+{
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, User::class);
+    }
+
+    /*
+    public function findBySomething($value)
```
Comment (Nyholm) [inline friendsofsymfony/user-bundle/2.1/src/Repository/UserRepository.php]: We should not use commented code like this

File: friendsofsymfony/user-bundle/2.1/src/Repository/UserRepository.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class UserRepository extends ServiceEntityRepository
```
Comment (Nyholm) [inline friendsofsymfony/user-bundle/2.1/src/Repository/UserRepository.php]: Do we really need a UserRepository

File: friendsofsymfony/user-bundle/2.1/src/Entity/User.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * @ORM\Table(name="fos_user")
+ */
+class User extends BaseUser
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    protected $id;
```
Comment (Nyholm) [inline friendsofsymfony/user-bundle/2.1/src/Entity/User.php]: Shouldn't it be private?

**Comment by Nyholm** [comment]
Any updates here?

**Comment by fabpot** [comment]
Closing as a duplicate (#270)

**Pull request closed without merging**